app.js

```diff
@@ -0,0 +1,131 @@
+// Prerequisite data SIP and Access Token(Hard Coded)
+const myAccessToken = "ODcwYWE1MDQtODc5ZC00ZDhiLWI5ODYtZTk0ZGUyNTA4YzM2MTIwNmIzOTQtZDhm_PF84_1eb65fdf-9643-417f-9974-ad72cae0e10f";

```

Comment:
This is a public repository. Let's avoid pushing any access tokens here-on. Not safe to do it. Please leave it empty and whoever uses shall fill theirs